wemake_python_styleguide/visitors/ast/attributes.py

```diff
@@ -73,3 +74,38 @@ def _check_magic_attribute(self, node: ast.Attribute) -> None:
                 self._ensure_attribute_type(
                     node, DirectMagicAttributeAccessViolation,
                 )
+
+
+@final
+class EncodingVisitor(BaseNodeVisitor):
+    """Check if open function has the encoding parameter."""
+
+    def __init__(self, *args, **kwargs) -> None:
+        """Creates the booleans indicating encoding was found."""
+        super().__init__(*args, **kwargs)
+        self._encoding = False
+
+    def visit_Call(self, node: ast.Call):
+        """Visit calls and finds if it is an open function."""
+        if isinstance(node.func, ast.Name) and node.func.id == 'open':
+            if node.keywords:
+                for keyword in node.keywords:
+                    self._check_keywords(keyword)
+            if node.args:
+                self._check_args(node.args)
+
+            if self._encoding is False:
+                self.add_violation(UnspecifiedEncodingViolation(node))
+
+        self.generic_visit(node)
+
+    def _check_keywords(self, node: ast.keyword):
+        """Check if there is an encoding parameter."""
+        if node.arg == 'encoding':
+            self._encoding = True
+
+    def _check_args(self, node: List[ast.expr]):
```

Comment:
Two things:
1. `-> None`
2. `node` is not a proper name here, let's use `positional_args` or `positionals`